.github/workflows/dockerhub.yml

```
@@ -0,0 +1,46 @@
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Extract Git information
+      id: git-info
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:26: Double quote to prevent globbing and word splitting

(shellcheck)